plugins/api/src/experimental_api_plugin_adaptor.cpp

```diff
@@ -0,0 +1,130 @@
+#include "apiHandler.hpp"
+#include "experimental_plugin_framework.hpp"
+#include "client_api_whitelist.hpp"
+#include "json.hpp"
+
+#ifdef RODS_SERVER
+#include "plugin_lifetime_manager.hpp"
+#endif
+
+namespace {
+    namespace ixa = irods::experimental::api;
+
+    auto to_json(bytesBuf_t* b)
+    {
+        std::string foo{(char*)b->buf, (char*)b->buf+b->len};
+        return json::parse({(char*)b->buf, (char*)b->buf+b->len});
+    } // to_json
+
+
+
+    auto to_bbuf(const json& j)
+    {
+        auto s = j.dump();
+        auto b = new bytesBuf_t;
+        b->buf = new char[s.size()+1];
```

Comment:
will these be returned through the framework?  if so, shouldn't `std::malloc` be used?